packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -89,6 +89,23 @@ List<double> computeRenderSizes({
     renderSizes = transformToRenderSize(largestRenderSize);
   }
   return renderSizes;
+
+  ScrollController linkedController;
+  List<dynamic> nodes;
+  ListView.builder(
+    scrollDirection: Axis.horizontal,
+    controller: linkedController,
+    itemBuilder: (context, index) {
+      final node = nodes[index];
+      final next = index == nodes.length - 1 ? null : nodes[index + 1];
+      return Padding(
+        padding:
+            EdgeInsets.only(right: next == null ? 0.0 : next.left - node.right),
+        child: node.buildWidget(context, node.selected),
+      );
+    },
+    itemCount: nodes.length,
```

Comment:
dead code after return?

Reply:
itemBuilder is a callback.

For clarity, I'll put that last so that all the fields on the constructor are grouped together without the callback in the way.

Reply:
wait I'm not following here, so this is fields for the `FlexLayoutProperties` right?

this code is placed after the return statement in the `computeRenderSizes()` function, will it still be executed after return?

Reply:
Oh I see, this got moved around during merging.  Fixed.